ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -1,464 +1,476 @@
 {# eos - router bgp #}
-{% if router_bgp is defined and router_bgp is not none %}
+{% if router_bgp is arista.avd.defined %}
 !
 router bgp {{ router_bgp.as }}
    router-id {{ router_bgp.router_id }}
-{% if router_bgp.bgp_cluster_id is defined %}
+{%     if router_bgp.bgp_cluster_id is arista.avd.defined %}
    bgp cluster-id {{ router_bgp.bgp_cluster_id }}
-{% endif %}
+{%     endif %}
 {%     for bgp_default in router_bgp.bgp_defaults %}
```

Comment:
add check for bgp_defaults. Can be implemented as if outside for loop or with `| arista.avd.default([])` on the for-loop

Reply:
agreed updated